packages/repository-tests/src/crud/relations/helpers.ts

```diff
@@ -21,14 +23,37 @@ import {
 export function givenBoundCrudRepositories(
   db: juggler.DataSource,
   repositoryClass: CrudRepositoryCtor,
+  features: CrudFeatures,
 ) {
+  // when running the test suite on MongoDB, we don't really need to setup
+  // this config for mongo connector to pass the test.
+  // however real-world applications might have such config for MongoDB
+  // setting it up to check if it works fine as well
+  Order.definition.properties.customerId.type = features.idType;
+  Order.definition.properties.customerId.mongodb = {
+    dataType: 'ObjectID',
+  };
+  Address.definition.properties.customerId.type = features.idType;
+  Address.definition.properties.customerId.mongodb = {
+    dataType: 'ObjectID',
+  };
   // get the repository class and create a new instance of it
   const customerRepoClass = createCustomerRepo(repositoryClass);
   const customerRepo: CustomerRepository = new customerRepoClass(
     db,
     async () => orderRepo,
     async () => addressRepo,
   );
+  // register the inclusionResolvers here for customerRepo
+
```

Comment:
Let's add the newline _before_ the comment. I feel the comment belongs to registration of inclusion resolvers below, not to `new customerRepoClass` code above.

```ts
  const customerRepo: CustomerRepository = new customerRepoClass(
    db,
    async () => orderRepo,
    async () => addressRepo,
  );

  // register the inclusionResolvers here for customerRepo
  customerRepo.inclusionResolvers.set(
    'orders',
    customerRepo.orders.inclusionResolver,
  );
```